WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -517,7 +517,7 @@ extension PublishingEditor {
 
         PostCoordinator.shared.save(post)
 
-        dismissOrPopView(presentBloggingReminders: FeatureFlag.bloggingReminders.enabled)
+        dismissOrPopView(presentBloggingReminders: !JetpackNotificationMigrationService.shared.shouldDisableWordPressNotifications())
```

Comment:
I think with this change if the Blogging Reminders feature flag is disabled this popup will still show up.
Would it be better to still verify the feature flag is enabled as well?

Reply:
This is how it looked before local notifications work, so it shouldn't create any regressions:
```dismissOrPopView(presentBloggingReminders: true)```